app/main.py

```diff
@@ -1,8 +1,55 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+        self, comfort_class: int, clean_mark: int, brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+        self,
+        distance_from_city_center: float,
+        clean_power: int,
+        average_rating: float,
+        count_of_ratings: int,
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = round(average_rating, 1)
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        if car.clean_mark >= self.clean_power:
+            return 0.0
+        calculate_price = round(
+            (
+                car.comfort_class
+                * (self.clean_power - car.clean_mark)
+                * (self.average_rating / self.distance_from_city_center)
+            ),
+            1,
+        )
+        return calculate_price
+
+    def wash_single_car(self, car: Car) -> None:
+        if car.clean_mark < self.clean_power:
+            car.clean_mark = self.clean_power
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        total_amount = 0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
+                price = self.calculate_washing_price(car)
+                total_amount += price
```

Comment:
```suggestion
                total_amount += self.calculate_washing_price(car)
```
and you can delete the `price` variable above. In general, the less code, the better)